AutoRest/AutoRest.Core/ClientModel/ServiceClient.cs

```diff
@@ -93,6 +94,11 @@ public IEnumerable<string> MethodGroups
         public Dictionary<string, object> Extensions { get; private set; }
 
         /// <summary>
+        /// Gets page extensions dictionary.
+        /// </summary>
+        public Dictionary<KeyValuePair<string, string>, string> PageClasses { get; private set; }
```

Comment:
We should either not have this property on ServiceClient or make x-ms-pageable not Azure specific.